book/src/Example.md

```diff
@@ -91,7 +93,7 @@ The CSI sidecar apps are going to connect to the CSI driver, therefore
 starting it first helps avoid timeouts and intermittent container
 restarts:
 
-> $ kubectl create -f [https://raw.githubusercontent.com/kubernetes/kubernetes/f40a5d1155aae95105a4e9bb8933d750c666e350/test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpathplugin.yaml](https://github.com/kubernetes/kubernetes/blob/f40a5d1155aae95105a4e9bb8933d750c666e350/test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpathplugin.yaml)
+> $ kubectl create -f [https://github.com/kubernetes/kubernetes/blob/1a3fbf1f13a28f3102edc43ab2dca95471453cd8/test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpathplugin.yaml](https://github.com/kubernetes/kubernetes/blob/1a3fbf1f13a28f3102edc43ab2dca95471453cd8/test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpathplugin.yaml)
```

Comment:
Instead of pulling from kubernetes e2e test manifests, should we pull from yamls in kubernetes-csi/drivers?

Reply:
That makes sense, but someone would first have to update the drivers repo such that it actually has up-to-date yaml files. Right now there aren't any.

Reply:
Let's do that first, so we don't have to update this twice?